### ISSUE TITLE
ImportError: cannot import name 'Auth' from 'github'

### ISSUE THREAD

**Issue by Antoine-Sevec** [PyGithub/PyGithub#2562]
Hi,
Sorry for my low level of English...

I'm testing your project, but I'm stuck on authentication.

Here is my code:
```python
from github import Auth
from github import Github

def Authenticate(url, token):
    try:
        auth = Auth.Token(token)
        return Github(auth=auth, base_url=url)
    except Exception as e:
        print(e)
```

Here is the error:
```shell
  File "/app/src/core/github/__init__.py", line 2, in <module>
    from github import Auth
ImportError: cannot import name 'Auth' from 'github' (/usr/local/lib/python3.10/site-packages/github/__init__.py)
```

Here are the installed versions:
```shell
[tool.poetry.dependencies]
python = "^3.10"
PyGithub = "^1.58.2"
```

Would you have an idea of the problem?
Thank you so much

**Comment by EnricoMi** [comment]
You are looking at documentation of unreleased code, please see here for documentation of 1.58.2: https://pygithub.readthedocs.io/en/stable/introduction.html

The new way of authentication will be released shortly as 1.59.0.

**Comment by Antoine-Sevec** [comment]
Ok I had clicked on the link available on the github which redirects me to the page: [https://pygithub.readthedocs.io/en/latest/introduction.html](https://pygithub.readthedocs.io/en/latest/introduction.html)

And not on your link: [https://pygithub.readthedocs.io/en/stable/introduction.html](https://pygithub.readthedocs.io/en/stable/introduction.html)
Thank you for your very quick response.

and we can close the issue :)
